src/app/stack/utils/report-summary-utils.ts

```diff
@@ -241,7 +271,7 @@ export class ReportSummaryUtils {
 
             let stackLicense: MReportSummaryInfoEntry = new MReportSummaryInfoEntry();
             stackLicense.infoText = 'Suggested License';
-            let stackLicenses = licenseAnalysis.f8a_stack_licenses;
+            let stackLicenses = licenseAnalysis.current_stack_license;
```

Comment:
This has to be changed to `recommended_license`.

See https://github.com/fabric8-analytics/f8a-server-backbone/pull/218.

Reply:
fixed